pom.xml

```diff
@@ -42,11 +42,13 @@
 
         <ahc.version>2.10.4</ahc.version>
         <camel.version>3.0.1</camel.version>
+        <jira.guava.version>20.0</jira.guava.version>
```

Comment:
Yeah, I don't think we want to have Guava in `dependencyManagement` within the BOM. That has the effect of forcing the Guava version for the other components, right?

Can we not just get away with adding it directly into the Jira extension runtime POM?

Reply:
> Yeah, I don't think we want to have Guava in `dependencyManagement` within the BOM. That has the effect of forcing the Guava version for the other components, right?

No, I am fine with managing it. Having one version for all is good IMO, because that ensures that any combination of extensions can coexist in a single application.

Reply:
OK, change the property to guava.version, and the version is 26.0-jre